src/components/Pages/Search/index.js

```diff
@@ -1,46 +1,160 @@
+    // const queryArr = decodeURIComponent(window.location.hash.split("#")[1]).split("&")
+    // searchedTerm = queryArr[0]
+    searchedTerm = decodeURIComponent(window.location.hash.split("#")[1])
+  }
+
+  // hook makes our input dynamic (useful for "as-you-type" filtering)
+  let [ searchString, setSearchString ] = useState(searchedTerm)
+
+  // Don't show pages without Url (there seems to be some pages that are 'live', but without a url - catch those here...)
+  const searchIndexWithUrl = searchIndex.filter( page => page.janisUrls.length > 0)
+
+  let [ searchResults, setSearchResults ] = useState(searchIndexWithUrl)
+
+  const searchButtonPressed = function() {
+    loader.start({
+      contentId: 'coa-search_results',
+      loaderId: 'coa-search_loading_wheel',
+    })
+    setTimeout(function(){
+      if (typeof window !== 'undefined') {
+        window.location.hash = searchString.toLocaleLowerCase()
+      }
+      const filteredSearch = searchWorker(searchIndexWithUrl, searchString)
+      setSearchResults(filteredSearch)
+      loader.end()
+    },1000)
+  }
+
+  const searchKeyInput = function(event) {
+    setSearchString(event.target.value)
+
+    // For Quick Search (no-delay)... use 👇this, instead of that 👆.
+    // const filteredSearch = searchWorker(searchIndexWithUrl, event.target.value)
+    // setSearchResults(filteredSearch)
+  }
 
   return (
     <div>
+
       <Head>
-        <title> {title} </title>
+        <title> {intl.formatMessage(i18n.search)} </title>
       </Head>
 
-      <PageHeader> {title} </PageHeader>
+      <PageHeader contentType={'search'}>
+        {intl.formatMessage(i18n.search)}
+        <div className="coa-search_bar_container">
+          <input
+            id="coa-search_input"
+            className="coa-search_inputs"
+            onChange={()=>searchKeyInput(event)}
+            value={searchString}
+          />
+
+          <button
+            className="coa-search_button"
+            onClick={()=>searchButtonPressed('click')}
+          >
+            {intl.formatMessage(i18n.search)}
+          </button>
+        </div>
+      </PageHeader>
+
+      <div id="coa-search_loading_wheel" className="coa-loading_wheel"></div>
+
+      <div className="wrapper container-fluid">
+        <div className="row">
+
+          <div id="coa-search_results" className="col-xs-12 col-md-8">
 
-      <br /><h1> &nbsp; &nbsp; 🚧... Under Construction 🏗</h1>
+            {searchedTerm && searchResults.length < 1 && (
+              <NoResults />
+            )}
 
-      {/* - - - TO DO: Finish Janis Search Page - - - - - */}
+            <div className="coa-search_results-total">
+              {searchedTerm && searchResults.length > 0 && (
+                <span>
+                  {searchResults && searchResults.length}
+                  &nbsp;{intl.formatMessage(i18n.results)} <em>"{searchedTerm}"</em>
```

Comment:
This might be a good place to use i18n formatting https://github.com/cityofaustin/janis/blob/2135d4dc70792d0a70d0a2fcb7ddd63e761de0b0/src/js/i18n/definitions.js#L547-L549

Reply:
Alright take a look at what I did here. Looks to work for spanish as well.